### PR TITLE
TMDM-13127 Error message will be shown when importing content to server failed.

### PR DESCRIPTION
**Pull request by sbliu**
https://jira.talendforge.org/browse/TMDM-13127

**What is the current behavior?** (You should also link to an open issue here)
1. Import data to server from Studio always alert message of successfully(before eclipse platform upgrade to 4.10).
2. Import content to server & export content from server does not work after eclipse platform upgraded to 4.10 .

**What is the new behavior?**
1. Import content to server & export content from server work after eclipse platform upgraded to 4.10 .
2. Import data to server from Studio failed will show give error message to user and log it.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java [hunk 1]
```diff
@@ -326,9 +328,19 @@ protected void importClusterContents(IProgressMonitor monitor) {
             return;
         }
 
+        private void checkFailureReport(InputStreamMerger manager) throws Exception {
+            Throwable lastReportedFailure = manager.getLastReportedFailure();
+            if(lastReportedFailure != null) {
```
Comment (YohannTalend) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java]: Some spaces are missing after the if statement

Comment (sbliu) [reply main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java]: done

File: main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java [hunk 2]
```diff
@@ -240,7 +240,7 @@ protected void importClusterContents(IProgressMonitor monitor) {
                 unmarshaller.setWhitespacePreserve(true);
                 unmarshaller.setMapping(mapping);
             } catch (Exception e) {
-                log.error(e.getLocalizedMessage(), e);
+                LOG.error("Prepare unmarshaller error", e);
```
Comment (VictorLi-talendbj) [inline main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java]: The error message might be change to `Failed to prepare unmarshaller.`?

Comment (sbliu) [reply main/plugins/org.talend.mdm.repository/src/main/java/org/talend/mdm/repository/core/datacontent/impl/ImportDataContentProcess.java]: done

**Pull request merged**